### ISSUE TITLE
Convert data 

### ISSUE THREAD

**Issue by UnknownNull** [ColinHDev/CPlot#91]
The .db format of the myplot plugin was not recognized to convert to a new database and could only recognize the world

**Comment by ColinHDev** [comment]
The `plots.db` file is in the `Server/plugin_data/MyPlot/` directory and MyPlot's `config.yml` `DataProvider` entry points to `sqlite`?

**Comment by UnknownNull** [comment]
Yes, I tested this method once and once again moved plots.db to plugin_data\cplot and both times I did not get results.

**Comment by ColinHDev** [comment]
And you are using the pm5 version of the plugin?

**Comment by UnknownNull** [comment]
yes

**Comment by Atara6431** [comment]
we all meet the problem!

**Comment by ColinHDev** [comment]
I tested the plugin with a MyPlot `.db` file, so there shouldn't be any problems. Could you provide your `.db` file, MyPlot `plugin_data` directory and world folder?

**Comment by UnknownNull** [comment]
https://www.mediafire.com/file/yi8qeqyxh9kyfue/plots.db/file

**Comment by ColinHDev** [comment]
Works for me

**Comment by UnknownNull** [comment]
Can you tell me which version and how it works well for you? So that I can test it like you

**Comment by ColinHDev** [comment]
That part of the code wasn't changed in a while, so the latest from the `development` branch should be fine

**Comment by UnknownNull** [comment]
> That part of the code wasn't changed in a while, so the latest from the `development` branch should be fine

The owners of the plots have changed...

**Comment by ColinHDev** [comment]
Could you please provide CPlot's `.sqlite` file?

**Comment by UnknownNull** [comment]
https://www.mediafire.com/file/dkcarm44yrr84qh/CPlot.sqlite/file

**Comment by UnknownNull** [comment]
and wtf?
https://cdn.discordapp.com/attachments/1113569932773425304/1129897191515902103/2023-07-16_01-23-39.mp4

**Comment by ColinHDev** [comment]
Sorry, but I'm currently in my exam phase, so I don't have that much time for the next few weeks to debug this.
You could try to use MySQL as data provider - but there is no guarantee that the bug doesn't exist there as well.

**Comment by UnknownNull** [comment]
Ok, good luck
Another thing I wanted to report is that by pressing /p home, it is transferred to another person's plot.
And I request that p homes command be added

> Sorry, but I'm currently in my exam phase, so I don't have that much time for the next few weeks to debug this. You could try to use MySQL as data provider - but there is no guarantee that the bug doesn't exist there as well.

**Comment by ColinHDev** [comment]
Could you try the latest version of the development branch to see if the error still occurs?

**Comment by ColinHDev** [comment]
And in case it doesn't work, please provide the `CPlot.sqlite` file again

**Comment by UnknownNull** [comment]
Apparently, the first bug that changed the names after warping and returning to the plot has been fixed

But there is still a problem with the /p home bug and resetting the plot and re-claiming it as posted in the video

https://www.mediafire.com/file/3uaoayxco062qc3/CPlot.sqlite/file

**Comment by UnknownNull** [comment]
Need more information?

**Comment by ColinHDev** [comment]
I've failed to reproduce this in any way, so if you have more information, it would be pretty helpful!

**Comment by 4rt2m1s** [comment]
Was there already a solution for this? I'm facing the same thing
